woodwork/tests/conftest.py

```diff
@@ -914,3 +914,48 @@ def outliers_df_spark(outliers_df_pandas):
 @pytest.fixture(params=["outliers_df_pandas", "outliers_df_dask", "outliers_df_spark"])
 def outliers_df(request):
     return request.getfixturevalue(request.param)
+
+
+@pytest.fixture()
+def timezones_df_pandas():
+    dtypes = {
+        "default_1": "datetime64[ns]",
+        "utc_1": "datetime64[ns, UTC]",
+        "eastern_1": "datetime64[ns, US/Eastern]",
+        "default_2": "object",
+        "utc_2": "object",
+        "eastern_2": "object",
+    }
+    return pd.DataFrame(
+        data=[
+            [
+                "2022-01-01",
+                "2022-01-01 00:00:00+00:00",
+                "2022-01-01 00:00:00-05:00",
+                "2022-01-01",
+                "2022-01-01T00:00:00+00:00",
+                "2022-01-01 00:00:00-05:00",
+            ],
+            [
+                "2022-01-02",
+                "2022-01-02 00:00:00+00:00",
+                "2022-01-02 00:00:00-05:00",
+                "2022-01-02",
+                "2022-01-02T00:00:00+00:00",
+                "2022-01-02 00:00:00-05:00",
```

Comment:
The last column here is supposed to be `Eastern`, right? Since there is the `-05:00` time? The column name calls it `eastern_2`, but the test that uses this checks that it's `UTC`. Can you explain which one it should be?

Reply:
Sure thing, so the original datetime string `00:00:00-05:00` contains an eastern timezone, but pandas cannot infer the timezone from the string and instead returns a fixed offset, so preserving the timezone is done by converting the datetime string to UTC `05:00:00+00:00`. Otherwise, the corresponding timezone gets lost after converting to `datetime64[ns]`. The test checks for the expected UTC conversion.

Reply:
Got it, thanks for explaining!